internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -295,6 +310,24 @@ func (r *AgentServiceConfigReconciler) ensureAssistedServiceDeployment(ctx conte
 	return nil
 }
 
+func (r *AgentServiceConfigReconciler) validateMirrorRegistriesConfigMap(ctx context.Context, instance *aiv1beta1.AgentServiceConfig) error {
+	mirrorCM := &corev1.ConfigMap{}
+	err := r.Get(ctx, types.NamespacedName{Name: instance.Spec.MirrorRegistryConfigMapName, Namespace: r.Namespace}, mirrorCM)
+	if err != nil {
+		r.Log.Info("Failed to get mirror registries ConfigMap")
+		return err
+	}
+	keysToValidate := []string{"ca-bundle.crt", "registries.conf"}
```

Comment:
```suggestion
	keysToValidate := []string{"ca-bundle.crt", MirrorRegistriesConfigFile}
```

Should `ca-bundle.crt` also be in a const?

Reply:
in this case those are the keys of the configmap, and it is used only here. i prefer to leave them as is, since the rest of the code also does not use const for other keys of the deployment

Reply:
As I see it, either we use constants that we've defined or we don't specify the constants.

The controller as it is now needs to get better at defining important values as constants and documenting them so I wouldn't use the current state as an argument to not define constants :sunglasses: .

From my perspective there are two options:
1. Define all of the important mirror registry related constants in this controller (or `common.go`); document them and use them everywhere those values (mostly strings) would appear. Here that would mean defining a constant for `"ca-bundle.crt"` and using it here.
2. Drop the constants and just refer to them everywhere explicitly as `"ca-bundle.crt"` or `"/etc/pki/ca-trust/extracted/pem/mirror-ca.pem"`

Reply:
fixed